File: src/game/events/NewToastGameEvent.java
package game.events;

import game.Bomberman;
import game.sound.AudioManager;

import static game.sound.SoundModel.BONUS_ALERT;

public class NewToastGameEvent implements GameEvent {
    @Override
    public void invoke(Object arg) {
        Bomberman.getBombermanFrame().getParentPanel().repaint();
        if((boolean) arg){
            AudioManager.getInstance().play(BONUS_ALERT);
        }
    }
}

File: src/game/engine/events/game/NewToastGameEvent.kt
package game.engine.events.game

import game.Bomberman
import game.engine.events.models.GameEvent
import game.engine.sound.AudioManager
import game.engine.sound.SoundModel

class NewToastGameEvent : GameEvent {
    override fun invoke(arg: Any?) {
        Bomberman.getBombermanFrame().parentPanel.repaint()
        if (arg as Boolean) {
            AudioManager.getInstance().play(SoundModel.BONUS_ALERT)
        }
    }
}